logstage/logstage-sink-file/src/main/scala/com/github/pshirshov/izumi/models/package.scala

```diff
@@ -0,0 +1,17 @@
+package com.github.pshirshov.izumi
+
+import com.github.pshirshov.izumi.models.LogFile.{DummyFile, RealFile}
+
+package object models {
+  type FileId = Int
+  type Files = scala.collection.mutable.Map[FileId, RealFile]
+
+  object FileId {
+    final val init = 0
+  }
+
+  object FileSize {
+    final val undefined = -1
```

Comment:
Consider using Option, or a separate sealed trait hierarchy instead of special constants